### ISSUE TITLE
Adding to vignette how to add the title to transmission chain plot

### ISSUE THREAD

**Issue by pbkeating** [reconhub/epicontacts#102]
Also, it might be worth adding in the vignette how to add a title to the transmission chain plan when plotting double x-axis based on date of onset or otherwise

I did the following:

```r
plot_dec$x$main$text <- "Chaînes de transmission connues, Août-Décembre 2018"
plot_dec$x$main$style <- "font-weight:bold;font-size:20px;text-align:center"
```

Obtained the info from here

datastorm-open/visNetwork#191

**Comment by zkamvar** [comment]
Thanks, @pbkeating! 

Another side note: if you want to paste several lines of code, you can use a code fence instead of single backtics. For example, I edited the above code like this:

    ```r
    plot_dec$x$main$text <- "Chaînes de transmission connues, Août-Décembre 2018"
    plot_dec$x$main$style <- "font-weight:bold;font-size:20px;text-align:center"
    ```

**Comment by finlaycampbell** [comment]
Thanks for this Pat! I've now added a new argument `title` that wraps this code and makes it available directly from a plot call (see 41666a2eff4d0c106281f7a89e664e825a42a12e)
